CHANGES.rst

```diff
@@ -51,6 +51,9 @@ general
 - Refactor general step input handling to avoid early closing of
   input files to allow using more lazy loading [#1342]
 
+- Provide suffix in Step.spec for steps to allow default suffix use
+  within and outside Pipeline runs [#1347]
+
```

Comment:
```suggestion
```

now that #1375 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Provide suffix in Step.spec for steps to allow default suffix use within and outside Pipeline runs" > changes/1347.general.rst
```

(this is just because #1375 was merged while active PRs existed; new PRs will have these instructions in their checklist)